### DISCUSSION TITLE
Not close after midi created

### DISCUSSION THREAD

**Discussion by jamesdeluk** [svsdval/video2midi#28] [Ideas]
Currently, after pressing `q` and the midi is created, the application closes.

It would be better if it stays open with the existing settings (i.e. the same as before `q` was pressed).

This way you can create a midi, check it, and then adjust the settings if you need to, and then recreate the midi, without having to start from scratch with key positioning etc.

**Comment by svsdval** [comment]
Hm, ok...

**Reply by svsdval** [reply]
Checkout ;)

**Reply by jamesdeluk** [reply]
Amazing!! And as an option too, even better. The video I've been trying to convert keeps missing a few notes (strangely only the left hand) and now I can tweak the settings easily until it's perfect (hopefully).

**Reply by svsdval** [reply]
Goodluck ;)
